hostile/1.1.0/Dockerfile

```diff
@@ -0,0 +1,71 @@
+# Set global variables
+ARG HOSTILE_VER="1.1.0"
+
+# Stage 1: Build Dockerfile
+FROM ubuntu:focal AS builder
+ARG HOSTILE_VER
+
+# Metadata
+LABEL base.image="ubuntu:focal" 
+LABEL dockerfile.version="1" 
+LABEL software="hostile"
+LABEL software.version="${HOSTILE_VER}"
+LABEL description="Precise host read removal."
+LABEL website="https://github.com/bede/hostile"
+LABEL license.url="https://github.com/bede/hostile?tab=MIT-1-ov-file#readme"
+LABEL maintainer="Taylor K. Paisie"
+LABEL maintainer.email="ltj8@cdc.gov"
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    software-properties-common && \
+    add-apt-repository ppa:deadsnakes/ppa && \
+    apt-get update && apt-get install -y --no-install-recommends \
+    python3.10 python3.10-distutils build-essential \
+    wget unzip samtools minimap2 bedtools gawk ca-certificates && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+
+
+# Install pip for Python 3.10
+RUN wget https://bootstrap.pypa.io/get-pip.py && python3.10 get-pip.py && \
+    rm get-pip.py && pip install --no-cache-dir setuptools pytest && \
+    ln -sf /usr/bin/python3.10 /usr/bin/python3 && \
+    ln -sf /usr/local/bin/pip /usr/bin/pip
+
+# Install bowtie2 and ensure all executables are accessible
+RUN wget https://github.com/BenLangmead/bowtie2/releases/download/v2.5.1/bowtie2-2.5.1-linux-x86_64.zip && \
```

Comment:
Can you adjust the bowtie2 version to an arg? Something like BOWTIE2_VER like the HOSTILE_VER arg.